pkg/notes/notes.go

```diff
@@ -422,9 +422,9 @@ func (g *Gatherer) ReleaseNoteFromCommit(result *Result) (*ReleaseNote, error) {
 	documentation := DocumentationFromString(prBody)
 
 	author := pr.GetUser().GetLogin()
-	authorURL := fmt.Sprintf("https://github.com/%s", author)
+	authorURL := fmt.Sprintf("%s/%s", g.options.GithubBaseURL, author)
 	prURL := fmt.Sprintf(
-		"https://github.com/%s/%s/pull/%d",
+		"%s/%s/%s/pull/%d", g.options.GithubBaseURL,
 		g.options.GithubOrg, g.options.GithubRepo, pr.GetNumber(),
```

Comment:
`g.options.GithubBaseURL` is empty here, the URLs produced lack the default value

ie, you're getting:
`/kubernetes/kubernetes/pull/93626`
and you want:  `https://github.com/kubernetes/kubernetes/pull/93626`

Reply:
Thanks for looking the failure! I now use `pr.GetHTMLURL()` which works well.